src/orion/algo/evolution_es.py

```diff
@@ -152,10 +151,22 @@ def __init__(self, evolutiones, budgets, repetition_id, space):
         super(BracketEVES, self).__init__(evolutiones, budgets, repetition_id)
         self.eves = self.hyperband
         self.space = space
-        self._search_space_remove_fidelity = []
+        self.search_space_remove_fidelity = []
+
+        if evolutiones.mutate_attr:
```

Comment:
Minor syntax comment, `evolutiones` should be `evolution_es`.

Reply:
I have fixed them. Please check.